common/djangoapps/util/views.py

```diff
@@ -231,6 +232,8 @@ def _record_feedback_in_zendesk(
         group = zendesk_api.get_group(group_name)
         if group is not None:
             new_ticket['ticket']['group_id'] = group['id']
+    if support_email is not None:
```

Comment:
Yes, if you do not include the `recipient` key here in the Zendesk ticket creation API request, Zendesk will use the default replay email address when support users respond to tickets.